hack/install-database.sh

```diff
@@ -0,0 +1,107 @@
+#!/bin/bash
+# Copyright KubeArchive Authors
+# SPDX-License-Identifier: Apache-2.0
+
+set -o errexit
+set -o xtrace
+
+kubectl apply -f - <<EOF
+---
+apiVersion: v1
+kind: Namespace
+metadata:
+  name: databases
+---
+apiVersion: v1
+kind: PersistentVolumeClaim
+metadata:
+  labels:
+    app: postgresql
+  name: postgresql
+  namespace: databases
+spec:
+  accessModes:
+    - ReadWriteOnce
+  resources:
+    requests:
+      storage: 5Gi
+---
+apiVersion: v1
+kind: PersistentVolume
+metadata:
+  labels:
+    type: local
```

Comment:
The pods that aim to use local PVs need to be allocated where the PV lives.
This works well when this is deployed on a one-worker-node Kubernetes cluster, otherwise it could randomly fail.

I suggest to do one of the following:
* Add a comment stating that this should be only used in single-worker-node kubernetes cluster.
* Add some constraints to make this work for multinode clusters:
  * Create a `StorageClass` with the `volumeBindingMode: WaitForFirstConsumer`
  * Update the `Deployment ` of the database to include all the pods under the same node, something along these lines:
  
```yaml
      spec:
      affinity:
        podAffinity:
          requiredDuringSchedulingIgnoredDuringExecution:
          - labelSelector:
              matchExpressions:
              - key: app
                operator: In
                values: 
                - postgresql
            topologyKey: "kubernetes.io/hostname"
```

Reply:
I don't think going the extra mile with the affinity (and its testing) is worth it as this is intended for CI, so I added a comment at the top of the script.